tests/python/sdk/common.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import io
+import zipfile
+from pathlib import Path
+
+from cvat_sdk.core.proxies.jobs import Job
+from cvat_sdk.core.proxies.projects import Project
+from cvat_sdk.core.proxies.tasks import Task
+from cvat_sdk.core.proxies.types import Location
+
+from shared.utils.config import IMPORT_EXPORT_BUCKET_ID
+from shared.utils.s3 import S3Client
+from shared.utils.s3 import make_client as make_s3_client
+
+from .util import make_pbar
+
+ProjectOrTaskOrJob = Project | Task | Job
+
+
+class _TestDatasetExport:
+    def _test_export_locally(
+        self,
+        resource: ProjectOrTaskOrJob,
+        *,
+        file_path: Path,
+        **export_kwargs,
+    ):
+        pbar_out = io.StringIO()
+        pbar = make_pbar(file=pbar_out)
+        resource.export_dataset(pbar=pbar, **export_kwargs)
+        assert self.stdout.getvalue() == ""
+        assert "100%" in pbar_out.getvalue().strip("\r").split("\r")[-1]
+        assert file_path.is_file()
```

Comment:
It looks a little bit odd that file path is passed in the arguments and checked after the export, but is not passed into the export function. In `_test_export_to_cloud_storage` as well.